coordinator/src/main/scala/filodb.coordinator/NodeClusterActor.scala

```diff
@@ -48,32 +48,32 @@ object NodeClusterActor {
    * Results in a state change and shard assignment to existing and new nodes.
    * Initiates ingestion on each node.
    *
-   * @param ref the DatasetRef for the dataset defined in MetaStore to start ingesting
+   * @param dataset the dataset to start ingesting
    * @param resources the sharding and number of nodes for ingestion and querying
    * @param source the IngestionSource on each node.  Use noOpSource to not start ingestion and
    *               manually push records into NodeCoordinator.
    * @param storeConfig a StoreConfig for the MemStore.
    * @return DatasetVerified - meaning the dataset and columns are valid.  Does not mean ingestion is
    *                           setup on all nodes - for that, subscribe to ShardMapUpdate's
    */
-  final case class SetupDataset(ref: DatasetRef,
+  final case class SetupDataset(dataset: Dataset,
```

Comment:
Be sure to test serialization of this.  We had trouble with serializing the Dataset class before.  It might be preferable to serialize using dataset.toString

Reply:
SetupDataset is never transmitted over the wire anymore. We use it only during unit tests. I will add comments as such